src/pages/settings/Profile/ProfilePage.js

```diff
@@ -153,7 +151,15 @@ class ProfilePage extends Component {
      * @param {Object} avatar
      */
     updateAvatar(avatar) {
-        this.setState({avatar: _.isUndefined(avatar) ? {uri: ReportUtils.getDefaultAvatar(this.props.currentUserPersonalDetails.login)} : avatar, isAvatarChanged: true});
+        this.setState({avatar, isAvatarChanged: true});
+    }
+
+    /**
+     * Replaces the user's current avatar image with a default avatar.
+     */
+    removeAvatar() {
```

Comment:
NAB: Should we maybe rename this to `deleteAvatar` too to keep it consistent?

Reply:
I think I'm down with this :D